src/main/java/org/ow2/proactive/procci/rest/ComputeRest.java

```diff
@@ -121,14 +102,12 @@
     public ResponseEntity<ResourceRendering> getCompute(@PathVariable("id") String id) {
```

Comment:
Not required with Spring to wrap all response types with ResponseEntity. You can directly return your object and throw exceptions. If the object is returned, then the status code is assumed to be HttpStatus.OK in this case, and if an exception is raised, it is handled by Spring and the right status code set.